kolibri/core/errorreports/models.py

```diff
@@ -36,3 +45,50 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):
             return False
 
         return None
+
+
+class ErrorReports(models.Model):
+    FRONTEND = "frontend"
+    BACKEND = "backend"
+    POSSIBLE_ERRORS = [
+        (FRONTEND, "Frontend"),
+        (BACKEND, "Backend"),
+    ]
+
+    error_from = models.CharField(max_length=10, choices=POSSIBLE_ERRORS)
+    error_message = models.CharField(max_length=255)
+    traceback = models.TextField()
+    first_occurred = models.DateTimeField(default=timezone.now)
+    last_occurred = models.DateTimeField(default=timezone.now)
+    sent = models.BooleanField(default=False)
+    no_of_errors = models.IntegerField(default=1)
+
+    def __str__(self):
+        return f"{self.error_message} ({self.error_from})"
+
+    def mark_as_sent(self):
+        self.sent = True
+        self.save()
+
+    @classmethod
+    def insert_or_update_error(cls, error_from, error_message, traceback):
+        if not settings.DEVELOPER_MODE:
```

Comment:
Can we use `getattr(settings, "DEVELOPER_MODE", None)` instead based the issues we discussed?